webplugin/js/app/kommunicate-client.js

```diff
@@ -94,6 +94,8 @@ Kommunicate.client={
                     if (typeof callback == 'function') {
                         callback(response.data.value);
                     }
```

Comment:
Same check is present at 3 places, see if directly the status value can be passed to the function. And that function can internally check this part